docs/helm-broker/03-07-addons-docs.md

```diff
@@ -1,41 +1,56 @@
-
-During the provisioning process, the Helm Broker pushes the **addonRepositoryURL** variable into the chart. The **addonsRepositoryURL** points to your addon compressed to a `.tgz` file.
-
-### Documentation structure
-
-Deliver documentation for your addons in Markdown files with specified metadata. To learn more about the metadata and content of the Markdown files, read [this](/components/headless-cms/#details-markdown-documents) document.
+Using the Helm Broker, you can provide documentation for your addon and display it in the Console UI. There are two cases in which you may want to use this feature:
+- Provide documentation for your addon
+- Provide documentation for objects that appear after provisioning your addon
+
+>**NOTE:** Deliver documentation for your addons in Markdown files with specified metadata. To learn more about the metadata and content of the Markdown files, read [this](/components/headless-cms/#details-markdown-documents) document. For more information about currently supported types of the assets, read [this](/components/headless-cms/#overview-overview-headless-cms-in-kyma) document.
+
+## Provide documentation for your addon
+
+To provide documentation for your addon, create the `docs` folder inside it. Your `docs` folder must contain a [`meta.yaml`](#details-create-addons-docs-directory) file with metadata information about how documentation for the addon is uploaded. You can either provide your own documents or point to the external URL with the source documentation:
+
+<div tabs>
+  <details>
+  <summary>
+  Provide your own documentation
+  </summary>
+
+Store your documents and assets in the `docs` folder inside your addon. Each Markdown file represents a separate tab in the Console UI. The documents are ordered according to the **type** metadata of your Markdown documents. Point the **filter** parameter of your `meta.yaml` file to the location where your documentation is stored, for example the `docs` directory.
+
+  </details>
+  <details>
+  <summary>
+  Use external documentation
+  </summary>
+
+In the `meta.yaml` file, provide the **url** parameter with a value that points to the address where the documentation is stored.
+
+  </details>
+</div>
+
+## Provide documentation for objects
+
+To provide documentation for objects that appear after provisioning your addon, create the `docs.yaml` file inside the addon's chart. This file contains [ClusterDocsTopic](/components/headless-cms/#custom-resource-cluster-docs-topic) or [DocsTopic](/components/headless-cms/#custom-resource-docstopic) custom resources. Each ClusterDocsTopic or DocsTopic corresponds to a single object with the same ID as the name of the specified object. Your `docs.yaml` file can contain many ClusterDocsTopics or DocsTopics.
```

Comment:
a yaml file, the name `docs.yaml` is not mandatory

Reply:
I guess it's the same with `meta.yaml` and `docs` directory?